src/util/formatNumber.ts

```diff
@@ -1,11 +1,9 @@
-//@ts-nocheck
-
 /** Adds commas to a number */
 // TODO: Localize
-export const formatNumber = n => {
+export const formatNumber = (n: number) => {
   let s = '' // eslint-disable-line fp/no-let
   const digits = n.toString().split('')
-  digits.forEach((value, i) => {
+  digits.forEach((_, i) => {
```

Comment:
@raineorshine Have you considered using `toLocaleString` instead of this?

Reply:
Sure, I'm not super familiar with `toLocaleString`.

Reply:
It does the same job that this function is doing. You can check it out [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString)

Reply:
Cool! I created #682 to replace it.